genie-core/src/main/java/com/netflix/genie/core/services/impl/LocalJobKillServiceImpl.java

```diff
@@ -131,6 +146,15 @@ public void killJob(
                 // Windows, etc. May support later
                 throw new java.lang.UnsupportedOperationException("Genie isn't currently supported on this OS");
             }
+
+            // Write additional file with kill reason
+            try {
+                this.objectMapper.writeValue(
+                    new File(this.baseWorkingDir + "/" + id + "/genie/kill-reason"),
```

Comment:
I would make the name of this file a constant probably in `JobConstants` that way we can at least change it and it will change all over the code where we reference it and we don't have to track down strings.